content/docs/guides/crosswalk/kubernetes/aws/02-kubernetes-stacks/01-cluster-configuration.md

```diff
@@ -0,0 +1,672 @@
+    const cluster = new eks.Cluster(`${projectName}`, {
+        instanceRoles: [
+            aws.iam.Role.get("adminsIamRole", stdNodegroupIamRoleName),
+            aws.iam.Role.get("devsIamRole", perfNodegroupIamRoleName),
+        ],
+        roleMappings: [
+            {
+                roleArn: config.adminsIamRoleArn,
+                groups: ["system:masters"],
+                username: "pulumi:admins",
+            },
+            {
+                roleArn: config.devsIamRoleArn,
+                groups: ["pulumi:devs"],
+                username: "pulumi:alice",
+            },
+        ],
+        vpcId: config.vpcId,
+        publicSubnetIds: config.publicSubnetIds,
+        privateSubnetIds: config.privateSubnetIds,
+        storageClasses: {
+            "gp2": { type: "gp2", encrypted: true},
+            "sc1": { type: "sc1"}
+        },
+        nodeAssociatePublicIpAddress: false,
+        skipDefaultNodeGroup: true,
+        deployDashboard: false,
+        version: "1.14",
+        tags: {
+            "Project": "k8s-aws-cluster",
+            "Org": "pulumi",
+        },
+        clusterSecurityGroupTags: { "ClusterSecurityGroupTag": "true" },
+        nodeSecurityGroupTags: { "NodeSecurityGroupTag": "true" },
+        enabledClusterLogTypes: ["api", "audit", "authenticator", "controllerManager", "scheduler"],
+        // endpointPublicAccess: false,     // Requires bastion to access cluster API endpoint
+        // endpointPrivateAccess: true,     // Requires bastion to access cluster API endpoint
+    });
+    
+    // Export the cluster details.
+    export const kubeconfig = cluster.kubeconfig;
+    export const clusterName = cluster.core.cluster.name;
+    ```
+
+## Create the Worker Nodes
+
+Create new worker node groups equipped with:
+
+**Identity**
+
+  * IAM role instance profiles for each worker node group.
+
+**Networking**
+
+  * No public IP addresses assigned to node group workers.
+  * Configurable security groups and ingress rules.
+
+**Workers**
+
+  * An AMI for a specific version of Kubernetes to use against the control plane.
+  * Separate machine types by use-case.
+  * Workers placed in an Auto Scaling Group.
+  * Desired capacity, min, and max sizes of the node group.
+
+**General**
+
+  * Kubernetes [labels][k8s-labels] and [taints][k8s-taints] for worker node segmentation and scheduling.
+  * Tags for resources under cluster management.
+
+	```typescript
+	import * as eks from "@pulumi/eks";
+	
+    // Create a Standard node group of t2.medium workers.
+    const ngStandard = new eks.NodeGroup(`${projectName}-ng-standard`, {
+        cluster: cluster,
+        instanceProfile: new aws.iam.InstanceProfile("ng-standard-instanceProfile", {role: stdNodegroupIamRoleName}),
+        nodeAssociatePublicIpAddress: false,
+        nodeSecurityGroup: cluster.nodeSecurityGroup,
+        clusterIngressRule: cluster.eksClusterIngressRule,
+        amiId: "ami-0ca5998dc2c88e64b", // k8s v1.14.7 in us-west-2
```

Comment:
Can this be a query, rather than hard-coded AMI that is specific to a region?

Reply:
This is being tracked in https://github.com/pulumi/pulumi-eks/issues/258.